src/runtime/node/tty/write-stream.ts

```diff
@@ -0,0 +1,38 @@
+import type tty from "node:tty";
+import { Socket } from "node:net";
+
+export class WriteStream extends Socket implements tty.WriteStream {
+  clearLine(dir: tty.Direction, callback?: (() => void) | undefined) {
```

Comment:
Callbacks in this class (clearLine, clearScreenDown, cursorTo, moveCursor) should be called